config/rbac/role.yaml

```diff
@@ -26,6 +26,26 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - velero.io
+  resources:
+  - downloadrequests
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Since we didn't implement a `watch` in the controller, should we remove this line?

Reply:
The very first bullet on this issue is to clean this up: https://github.com/vmware-tanzu/velero/issues/2884

I'll be trying to avoid this from being generated too.